docs/primer_short.md

```diff
@@ -30,10 +30,10 @@ The data available in a TND ultimately reduces to a 2x2 table of counts:
 
 - Individuals are vaccinated ($V$) with probability $v$, or unvaccinated ($U$)
   - Here we assume perfect reporting about vaccine status; i.e., there is no one who is actually vaccinated who appears in the "not vaccinated" arm, nor vice versa
-- Each individual has the possibility of becoming exposed $E$, and then has further conditional probabilities of becoming infected $I$, symptomatic $S_I$, and then tested $T_I$
-  - Conditional probabilities depend on vaccination status. E.g., the probability that a vaccinated person will receive a test is $P[E|V] \times P[I|E,V] \times P[S_I|I,V] \times P[T_I|S_I,V]$.
+- Each individual has the possibility of becoming exposed $E$, and then has further conditional probabilities of becoming symptomatically infected $S_I$, and then tested $T_I$
```

Comment:
As we've discussed before, splitting S_I into two steps (infection, then progressing to symptoms) can help us derive more nuanced conditions for when TND is(n't) biased. But collapsing these two steps into one looks _much_ cleaner, so it's probably more appropriate for the short primer. I like it!

Reply:
It makes the widget *very* manageable

I also like it for emphasizing that we're looking at $\mathrm{VE}_{SP}$